src/Security/Items.cs

```diff
@@ -165,10 +165,9 @@ internal SecKeyChain (NativeHandle handle)
 				else
 					copy.LowlevelSetObject (CFBoolean.TrueHandle, SecItem.ReturnData);
 				
-				IntPtr ptr;
-				status = SecItem.SecItemCopyMatching (copy.Handle, out ptr);
+				status = SecItem.SecItemCopyMatching (copy.Handle, out IntPtr ptr);
```

Comment:
Just wondering, any reason you're not using `var`?

```suggestion
				status = SecItem.SecItemCopyMatching (copy.Handle, out var ptr);
```

Reply:
Generally, when the type isn't apparent, i.e. `out` parameters, non-descriptive property/method names, I tend to find the explicit type better for reading.

It gives me the context I need to mentally map without an IDE (i.e. on code review, unfamiliar codebases). It's also common enough that it has an [.editorconfig](https://github.com/dotnet/roslyn/blob/8bd10f1548db958b1983f3bd236fcc56aafae1be/.editorconfig#L202) setting.

Reply:
I'm okay with changing it everywhere, I guess `IntPtr ptr` is noise.

Reply:
> I'm okay with changing it everywhere, I guess `IntPtr ptr` is noise.

Ok, great :)